tests/spec/invalid.coffee

```diff
@@ -89,3 +89,24 @@ describe "Invalid case tests", ->
       ]).then(cb).catch(
         (err) -> throw err
       ).done (-> done()), done
+
+describe "Print Java error message", ->
```

Comment:
btw, I didn't expect you wrote integrated tests. I thought writing unit tests is sufficient.